test-data/unit/semanal-errors.test

```diff
@@ -940,14 +940,6 @@ from typing import Generic
 class A(Generic[int]): pass # E: Free type variable expected in Generic[...]
 [out]
 
-[case testInvalidTypeWithinNestedGenericClass]
-from typing import Generic, TypeVar
-T = TypeVar('T')
-class A(Generic[T]):
-    class B(Generic[T]): pass \
-          # E: Free type variable expected in Generic[...]
```

Comment:
Oh, yes, you're totally right.  According to PEP484 you can neither make the inner class generic in the same tvar, nor use the outer tvar in the inner class.  I'll adjust as necessary.